SQLite/databaseSchema.sql

```diff
@@ -20,6 +20,12 @@ CREATE TABLE user (
 create table notification(
     notifyid INTEGER PRIMARY KEY,
     userid Integer,
+    minMagnitude decimal(6, 4),
```

Comment:
We had discussed storing these element in a string. We'll discuss the option tomorrow